src/test/integration/redis/tests/test_redis.py

```diff
@@ -0,0 +1,45 @@
+import os
+import sys
+import time
+import unittest
+from testcontainers.redis import RedisContainer
+import subprocess
+
+from test.test_utils.spans_parser import SpansContainer
+
+
+class TestRedisSpans(unittest.TestCase):
+    def test_redis_instrumentation(self):
+        with RedisContainer("redis:latest") as redis_server:
+            example_path = os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), "../app/redis_example.py"
```

Comment:
trivial and not blocking: i'm a little uncomfortable with the name `redis_example`, even though it really doesn't matter in this context; i would prefer it referenced what the example does, like `redis-set-and-get` or something